change/@ni-nimble-components-8ca8762f-ad08-436d-980f-56d9f5ba9e39.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Disable table row hover styling while scrolling",
+  "packageName": "@ni/nimble-components",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
The current change do see to improve the mobile behavior too! https://github.com/ni/nimble/issues/2312#issuecomment-2273419492

While scrolling you no longer see the hover state, it just appears again a little after stopping. Feels like an improvement 👍